.eslintrc.json

```diff
@@ -163,7 +163,8 @@
         "no-shadow": 2,
         "no-undef-init": 2,
         "no-undef": 2,
-        "no-unused-vars": 2,
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": ["off"],
```

Comment:
Why off? This rule is correct, and work as expected as I know

Reply:
for me it does not work and show an error when I define service in the constructor